### PR TITLE
Have muting hide other notification preferences

### PR DESCRIPTION
**Pull request by akalin-keybase**
Also fix info panel stories.

### CODE REVIEW

File: shared/chat/conversation/info-panel/notifications/index.js [hunk 1]
```diff
@@ -33,6 +33,80 @@ const SaveStateComponent = ({saveState}) => {
   }
 }
 
+const unmutedNotificationPrefs = (props: Props) => [
+  <Checkbox
```
Comment (akalin-keybase) [inline shared/chat/conversation/info-panel/notifications/index.js]: didn't change anything here, just turned into array and added keys

(can we use fragments yet?)

Comment (chrisnojima) [reply shared/chat/conversation/info-panel/notifications/index.js]: Yes but you have to use react.fragment and not the shortcut <>

Comment (akalin-keybase) [reply shared/chat/conversation/info-panel/notifications/index.js]: Done, thanks!

File: shared/chat/conversation/info-panel/notifications/index.js [hunk 2]
```diff
@@ -133,6 +109,35 @@ export const Notifications = (props: Props) => (
     <Box style={styleSaveState}>
       <SaveStateComponent saveState={props.saveState} />
```
Comment (buoyad) [inline shared/chat/conversation/info-panel/notifications/index.js]: I think this should remain visible when muting, right?

Comment (akalin-keybase) [reply shared/chat/conversation/info-panel/notifications/index.js]: Oops, yes! Done.

**Pull request merged**